ext/mysqli/mysqli.stub.php

```diff
@@ -0,0 +1,548 @@
+
+    /** @return bool */
+    public function close();
+
+    /** @return bool */
+    public function commit(int $flags = -1, string $name = UNKNOWN);
+
+    /** @return mysqli|null|false */
+    public function connect(
+        ?string $host = null,
+        ?string $user = null,
+        ?string $password = null,
+        ?string $database = null,
+        ?int $port = null,
+        ?string $socket = null
+    );
+
+    /** @return bool */
+    public function dump_debug_info();
+
+    /** @return bool */
+    public function debug(string $debug_options);
+
+    /** @return object|null */
+    public function get_charset();
+
+    /** @return string|null */
+    public function get_client_info();
+
+    /** @return array */
+    public function get_connection_stats();
+
+    /** @return string|null */
+    public function get_server_info();
+
+    /** @return mysqli_warning|false */
+    public function get_warnings();
+
+    /** @return mysqli|false */
+    public function init();
+
+    /** @return bool */
+    public function kill(int $connection_id);
+
+    /** @return bool */
+    public function multi_query(string $query);
+
+    /** @return bool */
+    public function more_results();
+
+    /** @return bool */
+    public function next_result();
+
+    /** @return bool */
+    public function ping();
+
+    /** @return int|false */
+    public static function poll(?array &$read, ?array &$write, array &$error, int $sec, int $usec = 0);
+
+    /** @return mysqli_stmt|false */
+    public function prepare(string $query);
+
+    /** @return mysqli_result|bool */
+    public function query(string $query, int $resultmode = MYSQLI_STORE_RESULT);
+
+    /** @return mysqli|bool */
+    public function real_connect(
+        ?string $host = null,
+        ?string $user = null,
+        ?string $password = null,
+        ?string $database = null,
+        ?int $port = null,
+        ?string $socket = null,
+        int $flags = 0
+    );
+
+    /** @return string */
+    public function real_escape_string(string $string_to_escape);
+
+    /** @return mysqli_result|bool */
+    public function reap_async_query();
+
+    /** @return string */
+    public function escape_string(string $string_to_escape);
+
+    /** @return bool */
+    public function real_query(string $query);
+
+    /** @return bool */
+    public function release_savepoint(string $name);
+
+    /** @return bool */
+    public function rollback(?int $flags = 0, ?string $name = '');
+
+    /** @return bool */
+    public function savepoint(string $name);
+
+    /** @return bool */
+    public function select_db(string $database);
+
+    /** @return bool */
+    public function set_charset(string $charset);
+
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    public function options(int $option, $value);
+
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    public function set_opt(int $option, $value);
```

Comment:
it's an alias, and I think we don't define aliases (at least I didn't do).

Reply:
Removed

Reply:
Aliases are supposed to be part of the stubs as well. Sorry if I didn't notice that in other reviews.

Reply:
No problem, I'll fix the missing stubs in the coming days.

Reply:
no problem :-) reverted